notebooks/official/pipelines/get_started_with_hpt_pipeline_components.ipynb

```diff
@@ -919,49 +961,62 @@
       },
       "outputs": [],
       "source": [
-        "TRAIN_IMAGE = \"gcr.io/\" + PROJECT_ID + \"/horses_or_humans:v1\""
+        "TRAIN_IMAGE = (\n",
+        "    f\"{LOCATION}-docker.pkg.dev/{PROJECT_ID}/{REPO_NAME}/horse_human_hptune:latest\"\n",
+        ")"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "build_container:training"
+        "id": "a6ad28220418"
       },
       "source": [
-        "Next, build the container."
+        "Then, build the container and push it to Artifact Registry by running the following:"
```

Comment:
Should: "build the container and push it to Artifact Registry by running the following"
Be: "build the container and push it to an Artifact Registry repository by running the following".  ??